tests/foreman/cli/test_contentview.py

```diff
@@ -4023,6 +4023,34 @@ def test_positive_inc_update_no_lce(self, module_org, module_product, module_tar
         content_view = module_target_sat.cli.ContentView.info({'id': content_view['id']})
         assert '1.1' in [cvv_['version'] for cvv_ in content_view['versions']]
 
+    @pytest.mark.tier2
+    def test_version_info_by_lce(self, module_org, module_target_sat):
+        """Hammer version info can be passed the lce id/name argument without error
+
+        :id: 6ab0c46c-c62a-488b-a30f-5500d6c7ec96
+
+        :steps:
+            1. Lookup CV version info passing the lce id as an argument
+
+        :expectedresults: LCE is able to be passed to version info command without error
+
+        :BZ: 2139834
+
+        :customerscenario: true
+        """
+        content_view = module_target_sat.cli_factory.make_content_view(
+            {'organization-id': module_org.id}
+        )
+        module_target_sat.cli.ContentView.publish({'id': content_view['id']})
+        content_view = module_target_sat.cli.ContentView.version_info(
+            {'id': content_view['id'], 'lifecycle-environment-id': 1}
```

Comment:
I'm assuming the Library env is always ID = 1 but might be safe to also just do a search and make sure we are always getting the right ID instead of hard coding the ID. Also you could create a lifecycle env in the setup and use that ID / name.